apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
   annotations:
     serviceoperator.azure.com/reconcile-policy: detach-on-delete
 spec:
+  tags:
+    builtFrom: https://github.com/hmcts/cnp-flux-config
+    businessArea: CFT
+    environment: development
+    autoShutdown: "true"
+    application: social-service-child-support
```

Comment:
hmm this looks like a generic resource, it probably needs an environment variable being added into the environments post build setup.

e.g. in here:
https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/base/kustomize.yaml
adding an APPLICATION_TAG in there

Reply:
I have added this